source/dbo/Views/Power BI Reporting/vwNewAntibioticCodes.sql

```diff
@@ -0,0 +1,15 @@
+﻿CREATE VIEW [dbo].[vwNewAntibioticCodes]
+AS
+
+SELECT
+	su.LabAntibiotic,
+	a.ReferenceLaboratory,
+	COUNT(DISTINCT a.ReferenceLaboratoryNumber) AS NumberOfSamples,
+	MIN(a.AuditCreate) AS FirstIntroduced,
+	MAX(a.AuditCreate) AS LastSeen
+FROM [$(Labbase2)].dbo.Susceptibility su
+	INNER JOIN [$(Labbase2)].dbo.Anonymised a ON a.LabDataID = su.LabDataID
+WHERE su.AntibioticCode = 'New'
+	AND a.IsAtypicalOrganismRecord = 0
+	AND a.MergedRecord = 0
+GROUP BY su.LabAntibiotic, a.ReferenceLaboratory
```

Comment:
This wasn't specified in the ticket but it felt right to break out the missing antibiotic issue into two views: one where we are missing the mapping (the other view to this one), and one where it wasn't mapped properly during `LabBase2` import. These are a rough guess of what fields that will be useful when debugging this but let me know if you have any suggestions